### DISCUSSION TITLE
Is it possible to move issue to `Ungroomed` state?

### DISCUSSION THREAD

**Discussion by antosha417** [ankitpokhrel/jira-cli#723] [Q&A]
I'm trying to move an issue and getting an error. 
```
$ jira issue move ISSUE-685 "Ungroomed"

✗ Error: invalid transition state "Ungroomed"
Available states for issue ISSUE-685: 'To Do', 'In Progress', 'Done', 'Pending review', 'Rejected', 'Blocked', 'Progress Paused', 'Ready for deployment', 'Awaiting Decision', 'Waiting for Review', 'Dependency', 'In Design', 'Pending Design'
```

Is there a workaround to move issue to the Ungroomed state?

**Comment by rcoleman-hivewatch** [comment]
this smells like an issue with your jira workflow not supporting that "Ungroomed" state... not something with this `jira-cli` tool

**Reply by antosha417** [reply]
When I open jira in browser I can see "UNGROOMED". I can also select it and it transitions my issue to ungroomed state. 
<img width="278" alt="Screenshot 2024-03-20 at 19 13 21" src="https://github.com/ankitpokhrel/jira-cli/assets/14187674/e851d43b-5c28-45e9-aed1-a11bfa9add18">

**Reply by antosha417** [reply]
<img width="1625" alt="Screenshot 2024-03-20 at 19 14 59" src="https://github.com/ankitpokhrel/jira-cli/assets/14187674/bdd1e502-bd2b-45b2-8339-59474fd15357">
